### DISCUSSION TITLE
Oauth flow redirects me to an empty page

### DISCUSSION THREAD

**Discussion by bitbaggi** [directus/directus#3833] [Archive]
Hi Guys,

i'm running a directus/directus:v9.0.0-rc.24 docker container.
I tried to add my own single sign on provider to directus oauth feature.

Now i can see a button under the noraml login button.
When i click it directus redirects me only to http://localhost:8055/auth/oauth/keycloak?redirect=http://localhost:8055/admin/login with an empty page on it.

My Env Variables are:
OAUTH_KEYCLOAK_KEY: "client_id"
OAUTH_KEYCLOAK_SECRET: "client_id"
OAUTH_KEYCLOAK_SCOPE: "openid email"
OAUTH_KEYCLOAK_AUTHORIZE_URL: "https://sso-host.de/auth/realms/master/protocol/openid-connect/auth"
OAUTH_KEYCLOAK_ACCESS_URL: "https://sso-host.de/auth/realms/master/protocol/openid-connect/token"

Any ideas why it not redirects me to my given OAUTH_KEYCLOAK_AUTHORIZE_URL?

**Comment by meedkruu** [comment]
Hi,
I use the same docker container and try to use oauth for google and I have the same problem. Does anyone have an idea, how to fix that?
Br, Uta

**Reply by meedkruu** [reply]
Hi,
I found I had an error in my configuration, now it works, thanks :-)
Br, Uta
